addon/components/tag-panel.hbs

```diff
@@ -0,0 +1,109 @@
+<div class="tag-panel p-2 bg-none">
+  {{#if this.recentPostsTag}}
+    <div class="tag-panel-group">
+      <TagPostList @tag={{this.recentPostsTag}}></TagPostList>
+    </div>
+  {{/if}}
+
+  {{#if this.releasesTag}}
+    <div class="tag-panel-group">
+      <TagPostList @tag={{this.releasesTag}}></TagPostList>
+    </div>
+  {{/if}}
+
+  {{#if this.emberTimesTag}}
+    <div class="tag-panel-group">
+      <TagPostList @tag={{this.emberTimesTag}}></TagPostList>
+    </div>
+  {{/if}}
+
+ {{#if this.yearTags.length}}
+    <div class="tag-panel-group" data-test-year-group>
+      <h3
+        id="year-tag-group-title"
+        class="tag-panel-group-title text-lg"
+        data-test-tag-name
+      >
+        By Year
+      </h3>
+
+      <ul
+        class="tag-panel-inline-group list-unstyled ml-1"
+        aria-labelledby="year-tag-group-title"
+        data-test-tag-list
+      >
+        {{#each this.yearTags key="id" as |yearTag|}}
```

Comment:
you don't need to pass key here, it's automatic 👍

Reply:
Great catch 👍 Fixed!